src/thunderbots/software/ai/hl/stp/evaluation/deflect_off_enemy_target.cpp

```diff
@@ -0,0 +1,67 @@
+//
+// Created by roark on 02/02/19.
+//
+
+#include "ai/world/field.h"
+#include "ai/world/world.h"
+#include "software/geom/line.h"
+#include "software/geom/util.h"
+
+auto MAX_RADIUS = 0.15;
+namespace Evaluation
+{
+    Point deflect_off_enemy_target(World world)
+    {
+        Point enemy_goal_positive = world.field().enemyGoalpostPos();
+        Point enemy_goal_negative = world.field().enemyGoalpostNeg();
+
+        // The triangle formed by the enemy goalposts and the ball. Any robots in
+        // this triangle could block a chip/shot
+        Triangle chip_target_area =
+            triangle(world.ball().position(), enemy_goal_positive, enemy_goal_negative);
+
+        Robot enemyClosestToEdge = world.enemyTeam().getRobotById(0).value();
+        double shortestLenToEdge = 100.0;
+        double closestEdgeY;
+        // Finds the y value of the closest edge of the field (likely where the ball
+        // is)
+        if (world.ball().position().y() <= 0.0)
+            closestEdgeY = world.field().friendlyCornerNeg().y();
+        else
+            closestEdgeY = world.field().friendlyCornerPos().y();
+
+        // Find the enemy that's blocking a shot that's closest to the edge of the
+        // field
+        for (Robot i : world.enemyTeam().getAllRobots())
+        {
+            if ((contains(chip_target_area, i.position()) ||
+                 offsetToLine(enemy_goal_negative, world.ball().position(),
+                              i.position()) <= MAX_RADIUS ||
+                 offsetToLine(enemy_goal_positive, world.ball().position(),
+                              i.position()) <= MAX_RADIUS) &&
```

Comment:
These `MAX_RADIUS` values were previously the robot max radius. This value is defined in `shared/constants.h`, and should be used instead. (Applies for all usages of `MAX_RADIUS`)